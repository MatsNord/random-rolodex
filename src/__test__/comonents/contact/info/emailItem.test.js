import React from 'react';
import renderer from 'react-test-renderer';
import EmailItem from './../../../../components/contact/info/emailItem';

test('Email render', () => {

  const tree = renderer.create(<EmailItem />).toJSON();
  expect(tree).toMatchSnapshot();
});