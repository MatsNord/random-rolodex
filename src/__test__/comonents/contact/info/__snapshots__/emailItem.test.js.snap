// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Email render 1`] = `
<div
  className="info-item"
>
  <div
    className="info info-text"
  />
  <a
    href="mailto:undefined"
  />
</div>
`;
